component-lib/modal.html

```diff
@@ -0,0 +1,252 @@
+										class="btn btn-icon"
+										onclick="closeModel()"
+										class="closeModel"
+									>
+										<i class="fa fa-times fa-lg" aria-hidden="true"></i>
+									</button>
+								</div>
+								<div class="modal-body">
+									<p class="typo-subtext">
+										Model body goes here. The modal component provides a solid
+										foundation for creating dialogs, popovers, lightboxes, or
+										whatever else.
+									</p>
+								</div>
+								<div class="modal-footer">
+									<button class="btn btn-primary">Option1</button>
+									<button class="btn btn-outlined">Option2</button>
+								</div>
+							</div>
+						</div>
+						<div class="code-snippet-display">
+							<code>
+								<iframe
+									src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=solarized+dark&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522modal%2522%2520id%253D%2522modal%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522modal-heading%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522typo-title%2522%253EModal%2520Title%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520class%253D%2522btn%2520btn-icon%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520onclick%253D%2522closeModel%28%29%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520class%253D%2522closeModel%2522%250A%2520%2520%2520%2520%2520%2520%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-times%2520fa-lg%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522modal-body%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522typo-subtext%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Model%2520body%2520goes%2520here.%2520The%2520modal%2520component%2520provides%2520a%2520solid%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520foundation%2520for%2520creating%2520dialogs%252C%2520popovers%252C%2520lightboxes%252C%2520or%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520whatever%2520else.%250A%2520%2520%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522modal-footer%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EOption1%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-outlined%2522%253EOption2%253C%252Fbutton%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+									style="
+										width: 680px;
+										height: 652px;
+										border: 0;
+										transform: scale(1);
+										overflow: hidden;
+									"
```

Comment:
The inline styling could be removed

Reply:
Moved inline styles to the common CSS file in new commit.